package.json

```diff
@@ -33,7 +33,7 @@
     "@nestjs/swagger": "^5.2.1",
     "@polymeshassociation/hashicorp-vault-signing-manager": "1.1.1",
     "@polymeshassociation/local-signing-manager": "1.2.0",
-    "@polymeshassociation/polymesh-sdk": "^17.0.0",
+    "@polymeshassociation/polymesh-sdk": "18.0.0-beta.2",
```

Comment:
We should do a `develop` -> `master` before this one get merged.

We should probably use semantic release bot in this repo and get alpha + beta branches as its own task. I made one [here](https://polymesh.atlassian.net/browse/DA-438)